### PR TITLE
feat(github-comments): use org option

### PR DESCRIPTION
**Pull request by cathteng**
Use the OrganizationOption `sentry:github_pr_bot` in addition to the feature flag to determine if we should comment on the pull request tied to a suspect commit.

For ER-1587

### CODE REVIEW

File: src/sentry/tasks/integrations/github/pr_comment.py
Comment (EricHasegawa) [inline src/sentry/tasks/integrations/github/pr_comment.py:184]: Does this PR mean we can take out the TODO on line 183?

File: src/sentry/tasks/integrations/github/pr_comment.py
Comment (EricHasegawa) [inline src/sentry/tasks/integrations/github/pr_comment.py:185]: (nit): applying `not` to an `and` statement can be confusing since it's really an `or` statement now, could be more clear (but slightly more verbose) if you just do `not condition_1 or not condition_2`, your call

**Review by EricHasegawa — APPROVED** [review]
Non-blocking comments - looks good :)

File: src/sentry/tasks/integrations/github/pr_comment.py
Comment (cathteng) [reply src/sentry/tasks/integrations/github/pr_comment.py:185]: i like using `not (x and y)` because i think the logic is easier to follow in terms of the early return. like "if you don't meet both of these conditions, execute the code block"

File: src/sentry/tasks/integrations/github/pr_comment.py
Comment (cathteng) [reply src/sentry/tasks/integrations/github/pr_comment.py:184]: yes

**Review by AniketDas-Tekky — APPROVED** [review]
lgtm, let's test this E2E on your test org once this is merged

**Pull request merged**
